tsconfig.json

```diff
@@ -7,7 +7,6 @@
     "newLine": "LF",
     "sourceMap": true,
     "declaration": false,
-    "moduleResolution": "node",
```

Comment:
relates to mikro-orm `ConfigurationLoader` recent [issue](https://github.com/mikro-orm/mikro-orm/pull/5514).
relying on default for now to avoid `error TS5109: Option 'moduleResolution' must be set to 'NodeNext' (or left unspecified) when option 'module' is set to 'NodeNext'`